dags/acteurs/tasks/business_logic/replace_acteur_table.py

```diff
@@ -82,9 +53,56 @@ def replace_acteur_table(
         "propositionservice_sous_categories",
     ],
 ):
-    from django.db import connection
+    from django.db import DEFAULT_DB_ALIAS, connections
+
+    for table in tables:
+        copy_table_with_pg_tools(f"{prefix_dbt}{table}")
+
+    with connections[DEFAULT_DB_ALIAS].cursor() as cursor:
+        switch_tables(cursor, prefix_django, prefix_dbt, tables)
+
+
+def copy_table_with_pg_tools(table):
+    from django.conf import settings
 
-    with connection.cursor() as cursor:
+    warehouse_connection = settings.DB_WAREHOUSE
+    qfdmo_connection = settings.DATABASE_URL
 
-        copy_tables(cursor, schema_django, prefix_dbt, schema_dbt, tables)
-        switch_tables(cursor, schema_django, prefix_django, prefix_dbt, tables)
+    logger.warning(f"Copying table {table} using pg_dump/pg_restore")
+    try:
+        with tempfile.NamedTemporaryFile(delete=False) as tmpfile:
```

Comment:
```suggestion
        with tempfile.NamedTemporaryFile() as tmpfile:
```

Pourquoi est-ce qu'on ne supprime pas la base à la fin ?